shrs_example/src/main.rs

```diff
@@ -144,9 +152,16 @@ fn main() {
     // =-=-= Readline =-=-=
     // Initialize readline with all of our components
 
+    let mut snips = Snippets::new(ExpandSnippet::Always);
+    snips.add(
+        "ls".to_string(),
+        SnippetInfo::new("ls --color=auto", Position::Command),
```

Comment:
this is also defined as an alias below, we should remove this definition from aliases to avoid confusion